README.md

```diff
@@ -6,6 +6,18 @@ Unofficial Ruby API to get data from The Blue Alliance.
 
 This API returns JSON-parsed data fetched from The Blue Alliance's API.
 
+## Installation
+
+Add `tba.rb` to your Gemfile.
+```ruby
+gem 'tba'
+```
+
+Or manualy install
```

Comment:
if you add it to the Gemfile and run `bundle install` shouldn't that install it for you?